### DISCUSSION TITLE
How to change the default database when using StandaloneServer

### DISCUSSION THREAD

**Discussion by CalvinKirs** [apache/dolphinscheduler#6614] [Q&A]
The embedded H2 database used by StandaloneServer by default is mainly for developers to debug easily and users can quickly experience it.

If you need to change the default database, you need to modify the source code, (usually we do not recommend this, maybe at this time you are more suitable to use the stand-alone deployment) 
Find org.apache.dolphinscheduler.server.StandaloneServer, and then comment out this line of code
```
startDatabase();
```
Then configure database-related information in datasource.properties.

**Comment by zhongjiajie** [comment]
But what if want to use mysql/postgresql in development env? I find out when I stop standalone server, all metadata in H2 will be gone. And I have to recreate them by myself.
Image we have a bug have to reproduce in dev env, I create a workflow by stop standalone server by accident. I have to recreate in after I restart standalone server.
I thinks maybe we should support multiple database in development env, WDTY
